src/build.ts

```diff
@@ -140,6 +141,20 @@ export async function build(
     effects.logger.log(cachePath);
   }
 
+  // Copy over the DuckDB extensions and create the DuckDB manifest.
+  for (const path of globalImports) {
+    if (path.startsWith("/_duckdb/")) {
+      const sourcePath = join(cacheRoot, path);
+      effects.output.write(`${faint("build")} ${path} ${faint("→")} `);
+      const contents = await readFile(sourcePath);
+      const hash = createHash("sha256").update(contents).digest("hex").slice(0, 8);
+      const alias = applyHash(path, hash);
+      aliases.set(path, alias);
+      await effects.writeFile(alias, contents);
+    }
+  }
+  const duckdb_manifest = await duckDBManifest(duckdb, {root, log: true, aliases});
```

Comment:
```suggestion
  const duckDbManifest = await getDuckDbManifest(duckdb, {root, log: true, aliases});
```

Style:

* Use lowerCamelCase instead of lower_snake_case
* Only capitalize the first letter or acronyms
* Use a verb to distinguish between a function that generates something and the noun result